app/views/reviews/_form.html.erb

```diff
@@ -26,10 +26,10 @@
     <%= form.text_field :date, id: :review_date %>
   </div>
 
-  <div class="field">
-    <%= form.label :restaurant_id %>
-    <%= collection_select(:restaurant, :restaurant_id, Restaurant.all, :id, :name, {:selected => params[:id]}) %>
-  </div>
+  <!--<div class="field">-->
```

Comment:
Usually, it's better to just remove dead code

Reply:
TRUE!